src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -218,6 +223,10 @@ namespace winrt::TerminalApp::implementation
                                      0.5f,
                                      nullptr);
                 }
+                if (shiftPressed && !debugTap)
```

Comment:
Shouldn't it be esle if? Are we allowing both split pane and new window on alt+shift click?

Reply:
You are right! Thanks for the catch, fixed